admin-cli/executor/server_config.go

```diff
@@ -20,26 +20,20 @@
 package executor
```

Comment:
Is this file modification related to this patch?

Reply:
I use `batchHTTP` api in this patch, so I remove the api into `utils` package